tests/main/driver/test_example_configs.py

```diff
@@ -28,6 +28,10 @@
     "baroclinic_c12_orch_cpu.yaml",
     "tropical_read_restart_fortran.yml",
     "tropicalcyclone_c128.yaml",
+    "baroclinic_c384_cpu.yaml",
+    "baroclinic_c384_gpu.yaml",
+    "baroclinic_c3072_cpu.yaml",
+    "baroclinic_c3072_gpu.yaml",
```

Comment:
Should these be included in the PR?

Reply:
actually, I think the yaml configs should be in the PR but I don't think we want to include such a high-res run in the test framework, and I'm not sure about the c384 ones either, actually

Reply:
@oelbert - this PR is going to a _benchmark_ branch for use when conducting procurements.

Reply:
Hah, I'd missed that, thanks for flagging it!